### ISSUE TITLE
possible pip3 install lark version conflict on RHEL8 machines

### ISSUE THREAD

**Issue by shanegoff** [hpc/charliecloud#1317]
When I `pip3 install --isolated --target=lib --no-cache-dir lark==0.11.3`
A tar is downloaded no matter what I do, which contains an egg installer and doesn't work for us.
But when I install a slightly newer version
`pip3 install --isolated --target=lib --no-cache-dir lark==0.12.0`
A `.whl` file is downloaded and it works

```
$ pip3 install --isolated --target=lib --no-cache-dir lark==0.11.3
Collecting lark==0.11.3
  Downloading https://files.pythonhosted.org/packages/e6/10/0359cf35c64004933c74cbf87e15dd5da311ff087580b15c41e0b254d2d3/lark-0.11.3.tar.gz (229kB)
    100% |████████████████████████████████| 235kB 9.2MB/s
Installing collected packages: lark
  Running setup.py install for lark ... done
Successfully installed lark-0.11.3
$ rm -rf lib/lark*
$ pip3 install --isolated --target=lib --no-cache-dir lark==0.12.0
Collecting lark==0.12.0
  Downloading https://files.pythonhosted.org/packages/cb/39/cef2ccdfd984ae3cf93878d050c1b7c9354dd9493ce83fd9bb33a41f7a33/lark-0.12.0-py2.py3-none-any.whl (103kB)
    100% |████████████████████████████████| 112kB 2.2MB/s
Installing collected packages: lark
Successfully installed lark-0.12.0
```

I'm not certain what features come with a newer version of lark, but assuming it won't hurt us we could just bump the version up.

**Comment by shanegoff** [comment]
I tested on an alma virtualbox. This time I made sure to install wheel using `pip3 install --user wheel` before trying to run `autogen.sh` and it seems to work just fine, even though it downloads the same tar.gz file.

I tried to reconfigure my python stuff on the offending machine. I get some other unrelated pip error, and when I change the install line to `python3 -m pip` instead of just `pip3` it works.

Works no problem on the alma VM with `pip3`. 
I think something in my dependencies is just a little off. I think this issue is safe to close until we get other reports of this being an issue. We might want to check for wheel before the breakage occurs, and we can possibly change the install line to `python3 -m pip --isolated install --target=lib --ignore-installed "lark==${lark_version}"` as it seems to save you from system installed dependency problems and shouldn't impact other distros.

Thoughts?
